smithy-typescript-ssdk-libs/server-common/src/validation/index.ts

```diff
@@ -170,5 +153,5 @@ export const generateValidationMessage = (failure: ValidationFailure): string =>
       suffix = "must have unique values";
```

Comment:
looks like we are missing tests for this case. can you add some?

Reply:
Added here, and do the protocol tests in https://github.com/awslabs/smithy/pull/1622.